AngularPart1/Components/EditTodo/editTodo.html

```diff
@@ -0,0 +1,13 @@
+<div ng-controller="editController">
```

Comment:
The tasks says: "Reuse the form above to edit an article by clicking on article title". It implies that you need to create a component (directive) for form.